### PR TITLE
#83 - limit number of filtering tags

### LINKED ISSUE #80: Validator allows empty elements in arrays
It seems stray `,' in csv file can create empty values inside arrays created with DataSanitizer. This should not be allowed, because it creates weird behavior like tags with no name.

I think simply skipping those values should be alright. In case of returned empty array DataValidator should raise correct exceptions.

### LINKED ISSUE #83: Limit number of filtering tags
Too many tags can break display of webpage and it can cause companies to overuse them to appear on every search.

We need to:

- [ ] Decide the most optimal number of maximum tags.
- [x] Implement new range validation option, that will replace expected count
- [x] Apply the new validation to the tags, and adjust existing validations.

### PR DESCRIPTION
**Pull request by plencka**
It should close #83. Also includes tiny fix to #80. So closes #80.

Currently tag limit is set to 4.

### CODE REVIEW

**Review by kamilpiech97 — DISMISSED** [review]

File: backend/Exceptions/Validation/InvalidRangeValidationException.php [hunk 1]
```diff
@@ -6,12 +6,13 @@
 
 use Throwable;
 
-class InvalidCountValidationException extends ValidationException
+class InvalidRangeValidationException extends ValidationException
 {
     public function __construct(
         int $entryID,
```
Comment (EwelinaLasowy) [inline backend/Exceptions/Validation/InvalidRangeValidationException.php:12]: We use camelCase, don't we? So it should be `$entryId`.

**Review by EwelinaLasowy — CHANGES_REQUESTED** [review]

File: backend/Exceptions/Validation/InvalidRangeValidationException.php (hunk 1 above)
Comment (EwelinaLasowy) [inline backend/Exceptions/Validation/InvalidRangeValidationException.php:12]: We use camelCase, don't we? S it should be `entryId`.

**Review by EwelinaLasowy — CHANGES_REQUESTED** [review]

**Review by plencka — COMMENTED** [review]
Trying to fix github bug.

File: backend/Exceptions/Validation/InvalidRangeValidationException.php (hunk 1 above)
Comment (plencka) [reply backend/Exceptions/Validation/InvalidRangeValidationException.php:12]: Yes! I still didn't manage to track each of these, and I didn't figure out yet how to find stuff using regex/something similar.

**Review by EwelinaLasowy — APPROVED** [review]

File: backend/Exceptions/Validation/InvalidRangeValidationException.php (hunk 1 above)
Comment (mtracz) [reply backend/Exceptions/Validation/InvalidRangeValidationException.php]: In PHPStorm: https://www.jetbrains.com/help/phpstorm/searching-everywhere.html

**Review by mtracz — APPROVED** [review]

**Review by kamilpiech97 — APPROVED** [review]

**Pull request merged**